src/GRA.Web/Areas/MissionControl/Views/Mail/Detail.cshtml

```diff
@@ -1,14 +1,16 @@
 ﻿@model GRA.Controllers.ViewModel.MissionControl.Mail.MailDetailViewModel
 
+@inject Microsoft.Extensions.Localization.IStringLocalizer<GRA.Resources.Shared> SharedLocalizer
```

Comment:
Localization isn't needed on the MissionControl side of things.

Reply:
Fixed in commit a644d762f46ed05c11f0f643e52b135003a8f354